jbot/src/main/java/me/ramswaroop/jbot/core/slack/Bot.java

```diff
@@ -169,6 +169,8 @@ public final void handleTextMessage(WebSocketSession session, TextMessage textMe
             if (event.getType() != null) {
                 if (event.getType().equalsIgnoreCase(EventType.IM_OPEN.name())) {
                     slackService.addDmChannel(event.getChannelId());
+                } else if (event.getType().equalsIgnoreCase(EventType.IM_CREATED.name())) {
+                    slackService.addDmChannel(event.getChannel().getId());
```

Comment:
for some reason on that event type `im_created` the channelId is null at the top level of the object, and the channel property is populated with all its properties, that's why I went to get the channel and then the Id.